recipes/gnuastro/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gnuastro" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{name}}/{{name}}-{{version}}.tar.gz
+  sha256: 6d56dc161ecd80c64dbf09808f58114b52ad2ce648c71551b871f5d10d4c07c5
+
+build:
+  number: 0
+  skip:  True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - libtool
+  host:
+    - gsl
+    - cfitsio
+    - wcslib
+    - jpeg
+    - libtiff
+    - libgit2
+    - ghostscript
+  run:
+    - wcslib
+    - libcurl
+    - libcblas
+    - libgit2
+    - xz
+    - zlib
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgnuastro${SHLIB_EXT}  # [unix]
+    - test -f ${PREFIX}/lib/libgnuastro.a  # [unix]
```

Comment:
Are there any command line apps you can test? Even if it's just a simple `program_name --help`

Reply:
Thanks! I had it included in https://github.com/conda-forge/staged-recipes/pull/9723/commits/16656e2fe64240c184f8effe03517401ebac1626 but @mohammad-akhlaghi suggested me to remove those and use `make check` instead, since the CLI is tested according to the result of the `./configure`

Reply:
It's nice to check a second time as it tests that the relocation as worked as these tests are ran in a separate environment to the build.

Reply:
Thanks, will do.